dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -116,3 +116,22 @@ object reflects the newly requested size in `PersistentVolume.Spec.Capacity`. At
 this point, you can create or re-create a new pod from the PVC to finish the
 file system resizing. Once the pod is running, the newly requested size is
 available and `FileSystemResizePending` condition is removed from the PVC.
+
+[[recover_from_resize_failure]]
+== Recovering from Failure when Expanding Volumes
+
+If expanding underlying storage fails either on master or node,
+{product-title} admin can manually recover the PVC state and cancel the resize requests that are
+continuously retried by the controller without administrator intervention.
+
+Currently, this can be done manually by completing the following steps:
+
+1. Mark the PV that is bound to the claim (PVC) with the `Retain` reclaim policy.
+   This can be done by editing the PV and changing persistentVolumeReclaimPolicy to `Retain`.
+2. Delete the PVC, so as we can re-create it later.
```

Comment:
How about before deleting PVC, using `oc export pvc <pvcname> > pvc.yaml` to backup PVC, then the next we can using this `pvc.yaml` file to create a new PVC?

Reply:
That is a good idea.

Reply:
wait.. this may be problematic too because a PVC may also contain state that we do not want to persist in yaml and recreate (such as a pending resize condition).